.github/workflows/main.yml

```diff
@@ -49,6 +49,7 @@ jobs:
           mix deps.unlock --check-unused
           mix deps.audit
           npm audit --audit-level=moderate --prefix assets
+          npm audit fix
```

Comment:
I think you should run this manually, not in CI.

Reply:
rookie mistake 😢